src/sw.test.js

```diff
@@ -58,24 +50,39 @@ describe('Service Worker', () => {
         expect(fetchSpy).toHaveBeenCalledWith(expectedUrl, { mode: 'no-cors' });
       });
     });
+    describe('image requested when webp supported', () => {
```

Comment:
Can we rename this test to more accurately describe exactly what's happening - 
```suggestion
    describe('image not requested when image ends with .webp and webp supported', () => {
```